media/static-bundles.json

```diff
@@ -944,6 +949,19 @@
       ],
       "name": "firefox_whatsnew_88_en"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js"
```

Comment:
Should this bundle also include the scripts so we get FxA flow params / analytics data on the button clicks? 

```
"js/base/mozilla-fxa-product-button.js",
"js/base/mozilla-fxa-product-button-init.js"
```